modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_9_1/DDMStructureUpgradeProcess.java

```diff
@@ -107,9 +110,13 @@ private void _upgradeDDMStructure() throws Exception {
 
 					preparedStatement2.setString(1, definition);
 
+					long ctCollectionId = resultSet.getLong("ctCollectionId");
+
+					preparedStatement2.setLong(2, ctCollectionId);
```

Comment:
We should not declare a variable to use only once.
Can you inline it, please? 

I know the old code was doing it, so can you inline `definition` and `structureId` too?

Reply:
On my way to do it :)